### PR TITLE
DP-4858 Remove "DRAFT" from decision page

### PR DESCRIPTION
**Pull request by labbydev**
## Description

Remove the draft indicator from the top of the decision page

## Related Issue / Ticket
https://jira.state.ma.us/browse/DP-4858

## Steps to Test

1. Navigate to the decision page (http://localhost:3000/?p=pages-board-decision)
1. Observe you do not see "DRAFT" at the top of the page

## Screenshots
N/A

## Additional Notes:
N/A

#### Impacted Areas in Application

* Board Decision Page

#### @TODO

N/A

### CODE REVIEW

File: styleguide/source/_patterns/05-pages/board-decision.json [hunk 1]
```diff
@@ -15,9 +15,6 @@
       }]
     },
     "socialLinks": null,
-    "publishState": {
-      "text": "Draft"
-    },
```
Comment (gender404) [inline styleguide/source/_patterns/05-pages/board-decision.json:20]: should be `publishState: null`.  I prefer to leave the variable in there.

Comment (gender404) [reply styleguide/source/_patterns/05-pages/board-decision.json:20]: FYI objects { ... } => null whereas strings and numbers => ""

Comment (labbydev) [reply styleguide/source/_patterns/05-pages/board-decision.json:20]: @legostud can you take a look at my latest commit? For some reason the file appears the same here.

File: styleguide/source/_patterns/05-pages/board-decision.json [hunk 2]
```diff
@@ -15,9 +15,7 @@
       }]
     },
     "socialLinks": null,
-    "publishState": {
-      "text": "Draft"
-    },
+    "publishState": null,
```
Comment (gender404) [inline styleguide/source/_patterns/05-pages/board-decision.json:18]: Looks fine down here.

**Review by gender404 — APPROVED** [review]

**Comment by jesconstantine** [comment]
@labbydev thanks for your work here!  Please deploy this work to your github pages fork so that @cmajel can provide external / design review.  Thanks!

**Pull request merged**